packages/experiment-browser/src/config.ts

```diff
@@ -180,4 +186,5 @@ export const Defaults: ExperimentConfig = {
   analyticsProvider: null,
   exposureTrackingProvider: null,
   httpClient: FetchHttpClient,
+  deploymentKey: null,
```

Comment:
use undefined from here on out.

Reply:
`null` is a bad practice in javascript IMO. previous null values can stick around but moving forward always prefer undefined.